e2e/tests/ui/workspace-ui.spec.ts

```diff
@@ -1,189 +0,0 @@
-    // Date.parse returns 'NaN' if string is not a valid time. NaN is never equal to itself.
-    expect(lastChangedTime === lastChangedTime).toBe(true);
-
-    // Check snowman menu options contains expected options.
-    const snowmanMenu = await card.getCardSnowmanMenu();
-    const links = await snowmanMenu.getAllOptionTexts();
-    expect(links).toEqual(expect.arrayContaining(['Share', 'Edit', 'Duplicate', 'Delete']));
-  });
-
-  test('CANCEL in Edit page goes back to Your Workspaces page', async () => {
-    const workspaces = new WorkspacesPage(page);
-    await workspaces.load();
-
-    // Click Create New Workspace link on the Workspaces page.
-    const editPage = await workspaces.clickCreateNewWorkspace();
-
-    const dataAccessTierSelect = editPage.getDataAccessTierSelect();
-    expect(await dataAccessTierSelect.isDisabled()).toBe(false);
-
-    // 1000 characters is the limit in textarea.
-    const textAreaCharLimitElements = await page.$x(charLimitXpath);
-    for (const elm of textAreaCharLimitElements) {
-      const charLimitTexts = await getPropValue<string>(elm, 'textContent');
-      expect(charLimitTexts).toEqual('1000 characters remaining');
-    }
-
-    // Click Cancel button to exit Edit page.
-    const cancelButton = editPage.getCancelButton();
-    await cancelButton.clickAndWait();
-
-    // Back to Your Workspaces page.
-    await workspaces.waitForLoad();
-    expect(await workspaces.isLoaded()).toBe(true);
-  });
-
-  test('Workspaces display by access level filter', async () => {
-    const filter = ['Owner', 'Writer', 'Reader'];
-
-    const workspacesPage = new WorkspacesPage(page);
-    await workspacesPage.load();
-
-    const workspacesPageCards = await findAllCards(page);
-    if (workspacesPageCards.length === 0) {
-      return; // End test now, no Workspace card.
-    }
-
-    // Default Filter by Select menu value is 'All'.
-    const selectMenu = new ReactSelect(page, { name: 'Filter by' });
-    const defaultSelectedValue = await selectMenu.getSelectedOption();
-    expect(defaultSelectedValue).toEqual('All');
-
-    // Change Filter by value.
-    for (const accessLevel of filter) {
-      const filterSelectedValue = await workspacesPage.filterByAccessLevel(accessLevel);
-      expect(filterSelectedValue).toEqual(accessLevel); // Verify selected option
-      const cards = await findAllCards(page);
-      // If any card exists, get its Access Level and compare with filter level.
-      for (const card of cards) {
-        // Check one card only. There could be a lot of cards. We don't need to check every one.
-        const workspaceAccessLevel = await card.getAccessLevel();
-        expect(workspaceAccessLevel).toEqual(accessLevel.toUpperCase());
-        // Verify: Share, Edit, Duplicate and Delete actions.
-        await card.verifyWorkspaceCardMenuOptions(workspaceAccessLevel);
-        break;
-      }
-    }
-  });
-
-  test('Duplicate Workspace page', async () => {
```

Comment:
Curious if we are testing actual duplicate workspace flow anywhere

Reply:
I think we have some tests to cover various cases of duplicate workspace: https://github.com/all-of-us/workbench/blob/c60057bc5e9b5e0b61559e87d9b74b20186965db/ui/src/app/pages/workspace/workspace-edit.spec.tsx#L186

Reply:
This functionality is covered by another E2E test: `workspace-duplicate.spec.ts`, so I don't think we're losing any coverage by removing this one.